cpp/ql/test/library-tests/ir/ir/aliased_ssa_ir.expected

```diff
@@ -2268,40 +2275,43 @@ ir.cpp:
 #  519|     r0_4(glval<float>)   = VariableAddress[f]     : 
 #  519|     m0_5(float)          = InitializeParameter[f] : r0_4
 #  520|     r0_6(glval<int[3]>)  = VariableAddress[a1]    : 
-#  520|     r0_7(int)            = Constant[0]            : 
-#  520|     r0_8(glval<int>)     = PointerAdd             : r0_6, r0_7
-#  520|     r0_9(unknown[12])    = Constant[0]            : 
-#  520|     mu0_10(unknown[12])  = Store                  : r0_8, r0_9
-#  521|     r0_11(glval<int[3]>) = VariableAddress[a2]    : 
-#  521|     r0_12(int)           = Constant[0]            : 
-#  521|     r0_13(glval<int>)    = PointerAdd             : r0_11, r0_12
-#  521|     r0_14(glval<int>)    = VariableAddress[x]     : 
-#  521|     r0_15(int)           = Load                   : r0_14, m0_3
-#  521|     mu0_16(int)          = Store                  : r0_13, r0_15
-#  521|     r0_17(int)           = Constant[1]            : 
-#  521|     r0_18(glval<int>)    = PointerAdd             : r0_11, r0_17
-#  521|     r0_19(glval<float>)  = VariableAddress[f]     : 
-#  521|     r0_20(float)         = Load                   : r0_19, m0_5
-#  521|     r0_21(int)           = Convert                : r0_20
-#  521|     mu0_22(int)          = Store                  : r0_18, r0_21
-#  521|     r0_23(int)           = Constant[2]            : 
-#  521|     r0_24(glval<int>)    = PointerAdd             : r0_11, r0_23
-#  521|     r0_25(int)           = Constant[0]            : 
-#  521|     mu0_26(int)          = Store                  : r0_24, r0_25
-#  522|     r0_27(glval<int[3]>) = VariableAddress[a3]    : 
-#  522|     r0_28(int)           = Constant[0]            : 
-#  522|     r0_29(glval<int>)    = PointerAdd             : r0_27, r0_28
-#  522|     r0_30(glval<int>)    = VariableAddress[x]     : 
-#  522|     r0_31(int)           = Load                   : r0_30, m0_3
-#  522|     mu0_32(int)          = Store                  : r0_29, r0_31
-#  522|     r0_33(int)           = Constant[1]            : 
-#  522|     r0_34(glval<int>)    = PointerAdd             : r0_27, r0_33
-#  522|     r0_35(unknown[8])    = Constant[0]            : 
-#  522|     mu0_36(unknown[8])   = Store                  : r0_34, r0_35
-#  523|     v0_37(void)          = NoOp                   : 
-#  519|     v0_38(void)          = ReturnVoid             : 
-#  519|     v0_39(void)          = UnmodeledUse           : mu*
-#  519|     v0_40(void)          = ExitFunction           : 
+#  520|     mu0_7(int[3])        = Uninitialized          : r0_6
```

Comment:
Are there any tests that show the `Uninitialized` instruction being consumed by another instruction? For example, here it produces `mu0_7`, but I see no instruction with `mu0_7` as an operand.

Reply:
Not without the Chi node PR.  I only made this change in this PR because the primary purpose of my PR, the `ConditionDeclExpr` work, required refactoring the same code that @rdmarsh2 modified in his Chi PR. Making this change here saved him the trouble of a hard-to-resolve merge conflict.